momentGW/pbc/ints.py

```diff
@@ -65,6 +65,8 @@ def __init__(
         # Attributes
         self.kpts = kpts
         self._madelung = None
+        self._naux_full = None
+        self._naux = None
```

Comment:
what is `_naux`? the `naux` property doesn't use a private variable so it isn't needed right?

Reply:
It was introduced when creating the THC imports. I just realised that I had never defined it properly in the init, so added it now.